Anseys/metadata.txt

```diff
@@ -0,0 +1,33 @@
+publicationType : "book"
+publicationTitle : "Anseÿs de Gascogne"
+publicationDate : "2018"
+publicationPlace : "Paris"
+publicationSeries : "Classiques français du Moyen Âge"
+publicationNum: "184"
+publicationStemmaNum: "1"
+publicationAuthors : "Herbin, Jean-Charles; Triaud, Annie"
+publicationPage : "LVIII"
+publicationLink : ""
+workTitle : "Anseÿs de Gascogne"
+workViaf : "191522864"
+workOrigDate : "13th c."
```

Comment:
I think we should normalise the date (and include that in Guidelines). I'd be in favor of either a single date (if dated)

1201

or a range

1201-1300 (for «13th c.).

Reply:
Issue for that? Or @GusRiva 's opinion directly here?
1201 is a problem: what if it is really 1201? Range would be better…

Reply:
Yes, sorry, I meant : range in this case, single date only for unambiguously dated ms.